server/liveblog/themes/themes_assets/classic/views/posts.html

```diff
@@ -4,13 +4,52 @@
          ng-class="['lb-mainitem-{{ post.mainItem.item_type | varname }}', {'show-author-avatar': 
                         ctrl.timeline.settings.showAuthor &&
                         ctrl.timeline.settings.showAuthorAvatar &&
-                        ctrl.isAd(post),
+                        !ctrl.isAd(post),
                     'lb-post-permalink-selected': post._id === timeline.permalink._id}]">
-    <span ng-if="ctrl.isAd(post)">
+    <span ng-if="!ctrl.isAd(post)">
         <lb-author item="post.mainItem" timeline="timeline"></lb-author>
         <div class="lb-post-highlighted" ng-if="post.lb_highlight">
             <i class="icon-star icon--orange"></i>
         </div>
+
+        <div class="lb-post-permalink" style="right:30px" ng-click="showSocial = !showSocial">
+           
+            <i class="fa fa-share share-icon" title="{{ 'Social Share' | translate }}"></i>
+
+            <div ng-if="showSocial" class="share-box">
+
+                <a href="//www.facebook.com/sharer.php?s=100&p[title]=&p[url]={{ ctrl.timeline.permalink.get(post._id) }}" class='social facebook' target='_blank'
+                title="{{ 'Share this post on Facebook' | translate }}"
+                onclick="window.open(this.href,'targetWindow','toolbar=no,location=no,status=no,menubar=no,scrollbars=yes,resizable=yes,width=570,height=400');return false;">
```

Comment:
I would put this long onclick string in a variable instead of copying it and pasting it 5 times